tests/test_utils.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import pytest
+
+from tests.test_plugins import mocked_plugin_loader  # noqa: F401
```

Comment:
At this point, I think we should move it to pytest. But that's to be done separately, I'll add to #23.